### ISSUE TITLE
Wacatac.B!ml

### ISSUE THREAD

**Issue by olibobpri** [RedFireMRT84/Liinback-Patcher#5]
When trying to download the Windows Liinback Patcher, Windows Security found a threat called "Trojan:Win32/Wacatac.B!ml".

Is this genuine or a false positive?

Just want to make sure as I obviously don't want a trojan on my pc.

**Comment by osiem00** [comment]
Same problem

**Comment by ZomkaDEV** [comment]
It's a known false positive for all programs written in Go.
https://go.dev/doc/faq#virus

**Comment by olibobpri** [comment]
> It's a known false positive for all programs written in Go. https://go.dev/doc/faq#virus

ok, thanks!

**Comment by painthesk** [comment]
When I run a minecraft client, windows gives the same message and my game crashes, is there any way to fix that error, can someone help me to this ?

**Comment by ZomkaDEV** [comment]
> When I run a minecraft client, windows gives the same message and my game crashes, is there any way to fix that error, can someone help me to this ?

No. It’s a common false positive for programs compiled from high level languages like Python and Go. However, you shouldn’t be using this issue tracker for unrelated programs since it’s an issue tracker for Liinback Patcher. 

What Minecraft client are you using? I use Prism Launcher and have no issues.

**Comment by painthesk** [comment]
> > When I run a minecraft client, windows gives the same message and my game crashes, is there any way to fix that error, can someone help me to this ?
> 
> No. It’s a common false positive for programs compiled from high level languages like Python and Go. However, you shouldn’t be using this issue tracker for unrelated programs since it’s an issue tracker for Liinback Patcher.
> 
> What Minecraft client are you using? I use Prism Launcher and have no issues.

i using minecraft launcher, tl legacy and the prism launcher like u said, but it still crash game, should i turn down the firewall window ?

**Comment by ZomkaDEV** [comment]
I advise you against using TL Legacy and other clients that allow you to pirate Minecraft. Especially considering the shady history of TL Legacy of them embedding ads with inappropriate content or ones that promote casinos or P2W servers.

**Comment by painthesk** [comment]
> Tôi khuyên bạn không nên sử dụng TL Legacy và các ứng dụng khách khác cho phép bạn sao chép lậu Minecraft. Đặc biệt là khi xem xét lịch sử đen tối của TL Legacy khi họ nhúng quảng cáo có nội dung không phù hợp hoặc quảng cáo quảng bá sòng bạc hoặc máy chủ P2W.

but should i turn off firewall window ?
